README.md

```diff
@@ -14,51 +14,44 @@
 
 ## Pre-requisites
 
-1. Install Node.js from [installer](https://nodejs.org/en/)
-2. Or using [command line](https://nodejs.org/en/download/package-manager/)
+1. Install Node.js using [nvm](https://github.com/nvm-sh/nvm) (_RECOMMENDED_), or from [installer](https://nodejs.org/en/), or using [command line](https://nodejs.org/en/download/package-manager/)
 
 ## Build / Run
 
-1. Run `npm i` to install all node modules
-2. Start server using
+1. If Node was installed via `nvm`, run `nvm use` to use the correct version
+2. Run `npm i` to install all node modules
+3. Copy `.env.example`, rename to `.env`, and update any necessary config
+4. Start server using
 
 ```bash
-$ npm run start
+$ npm start
```

Comment:
npm has aliases for the following scripts in your package.json. `test, start, restart, and stop`. so for these scripts you do not need to specify run. you can just do `npm test` instead of `npm run test`, etc. 

here is some documentation on it: https://docs.npmjs.com/cli/run-script